doc/examples.rst

```diff
@@ -33,8 +33,13 @@ other sources.
    steering-gainsched
    steering-optimal
    kincar-flatsys
+<<<<<<< HEAD
+   mrac_siso_mit
+   mrac_siso_lyapunov
+=======
    mrac_siso_direct_mit_rule_statespace
    mrac_siso_direct_lya_rule_statespace
+>>>>>>> refs/remotes/origin/mrac-examples
```

Comment:
This is an artifact of a merge, which must be fixed

Reply:
Oops I was looking at an intermediate commit. Please rebase so that it does not appear in the commit history.

Reply:
In reference to your comment: Do a `git rebase -i main` and "squash" some commits.

Reply:
Thx, I will try `git rebase -i main`.

In steps 8 of the manual, (https://github.com/python-control/python-control/wiki/How-to-contribute-with-a-pull-request)
which branch must be updated & synchronized, main or feature?

Reply:
Both. First get your local main branch in sync with upstream and then rebase feature onto your synced local main